interfaces/policy/policy_test.go

```diff
@@ -1547,6 +1635,237 @@ plugs:
 	}
 }
 
+var (
+	model0 *asserts.Model
+	model1 *asserts.Model
+	model2 *asserts.Model
+)
+
+func init() {
+	a, err := asserts.Decode([]byte(`type: model
+authority-id: other-brand
+series: 16
+brand-id: other-brand
+model: other-model
```

Comment:
Could we call this my-model0 for consistency or is other-model already used and crucial for this test?

Reply:
no but that would be strange because it's not a my-brand model,  what could be possible is to call the variables myModel1, myModel2, and otherModel instead